.github/workflows/swift.yml

```
@@ -0,0 +1,15 @@
+name: Swift
+
+on: [push]
+
+jobs:
+  build:
```

We can also harden the job a bit, by preventing the child steps from having write access to the repository.

    permissions:
      contents: read
```suggestion
  build:
      permissions:
          contents: read
```

---

There's a minor syntax error that's failing the build.  Need `permissions:` e.g.
```
  build:
    permissions:
      contents: read
```